libc/test/src/sys/mman/linux/mlock_test.cpp

```diff
@@ -57,9 +81,12 @@ TEST(LlvmLibcMlockTest, UnMappedMemory) {
 TEST(LlvmLibcMlockTest, Overflow) {
   PageHolder holder;
   EXPECT_TRUE(holder.is_valid());
-  EXPECT_THAT(LIBC_NAMESPACE::mlock(holder.addr, -holder.size), Fails(EINVAL));
-  EXPECT_THAT(LIBC_NAMESPACE::munlock(holder.addr, -holder.size),
-              Fails(EINVAL));
+  size_t negative_size = -holder.size;
+  auto expected_errno = is_permitted_size(negative_size) ? EINVAL : ENOMEM;
```

Comment:
no `auto`

Reply:
https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable